### ISSUE TITLE
ci-cmd assumes that Athena is deployed to the same region with identity region

### ISSUE THREAD

**Issue by vmindru** [aws-samples/aws-cudos-framework-deployment#151]
During `CUDOS Dashboard`  deploy with `cid-cmd deploy` In the case where user has configured  identity region in different region with QuickSight region. 

e.g. I have my identity in `eu-central-1` but Athena is in `us-east-1` 

Following error will be thrown during QuickSight dataset creation 

```An error occurred (AccessDeniedException) when calling the ListUsers operation: Operation is being called from endpoint us-east-1, but your identity region is eu-central-1. Please use the eu-central-1 endpoint.```

extend log 

```
There are still 5 datasets missing: compute_savings_plan_eligible_spend, customer_all, ec2_running_cost, s3_view, summary_view
? Select AWS Athena database to use athenacurcfn_vmindru_cudos
unable to create, missing permissions: An error occurred (AccessDeniedException) when calling the ListUsers operation: Operation is being called from endpoint us-east-1, but your identity region is eu-central-1. Please use the eu-central-1 endpoint.
Creating dataset: customer_all...Checking if CUR is enabled and available...
        Athena table: vmindru_cudos
        Resource IDs: yes
        SavingsPlans: no
        Reserved Instances: no
done
unable to create, missing permissions: An error occurred (AccessDeniedException) when calling the ListUsers operation: Operation is being called from endpoint us-east-1, but your identity region is eu-central-1. Please use the eu-central-1 endpoint.
Creating dataset: ec2_running_cost...unable to create, missing permissions: An error occurred (AccessDeniedException) when calling the ListUsers operation: Operation is being called from endpoint us-east-1, but your identity region is eu-central-1. Please use the eu-central-1 endpoint.
Creating dataset: s3_view...unable to create, missing permissions: An error occurred (AccessDeniedException) when calling the ListUsers operation: Operation is being called from endpoint us-east-1, but your identity region is eu-central-1. Please use the eu-central-1 endpoint.
Creating dataset: summary_view...unable to create, missing permissions: An error occurred (AccessDeniedException) when calling the ListUsers operation: Operation is being called from endpoint us-east-1, but your identity region is eu-central-1. Please use the eu-central-1 endpoint.
```

```
Checking AWS environment...
        profile name: default
        accountId: 167515602781
        AWS userId: assumed-role/Admin/vmindru-Isengard
        Region: us-east-1
done
```

**Comment by vmindru** [comment]
For now maybe just add a point to  "Before you start, stating that all elements, S3 bucket, Athena, QuickSight, IdentityProvider have to be in the same region" :)

**Comment by darken99** [comment]
Fixed in #231
